nixos/modules/services/misc/evdevremapkeys.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.evdevremapkeys;
+
+in
+{
+  options.services.evdevremapkeys = {
+    enable = mkEnableOption (lib.mdDoc ''evdevremapkeys'');
+
+    config = mkOption {
```

Comment:
By convention we call this `settings`. Please remember.

Reply:
Thanks!